### PR TITLE
teemo: add recipe

### PR DESCRIPTION
**Pull request by toge**
Specify library name and version:  **teemo/2.7**

teemo is a download library.

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by uilianries — APPROVED** [review]
LGTM

File: recipes/teemo/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        if self.info.settings.compiler == "apple-clang" and Version(self.info.settings.compiler.version) < "12.0":
+            raise ConanInvalidConfiguration(f"{self.ref} can not build on apple-clang < 12.0.") 
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], destination=self.source_folder, strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["BUILD_TESTS"] = False
+        tc.variables["USE_STATIC_CRT "] = is_msvc_static_runtime(self)
```
Comment (prince-chrismc) [inline recipes/teemo/all/conanfile.py:64]: Typo?

```suggestion
        tc.variables["USE_STATIC_CRT"] = is_msvc_static_runtime(self)
```

**Review by uilianries — APPROVED** [review]
LGTM

**Comment by uilianries** [comment]
I just restarted it now, again.

**Review by jwillikers — APPROVED** [review]

**Pull request merged**